specs/BackgroundColor.md

```diff
@@ -0,0 +1,49 @@
+# Background
+WebView2 developers have provided feedback that there is a 'white flicker' when navigating between pages. This flicker comes from WebView briefly showing its default background color for when no web page is loaded. Developers should be able to set a custom background color for the WebView that matches the color scheme of their app and avoids this flicker. We have also received feedback requesting the ability to set the WebView's background color transparent. This way developers can create a seamless UI experience where the WebView displays web content directly over host app content. The BackgroundColor API addresses this need.
```

Comment:
If you set BackgroundColor to transparent and make some of your web content transparent will that work, or is it only for the loading screens scenarios?

Reply:
Setting BackgroundColor transparent and having no background elements/style properties in the webcontents will work.